ui/src/app/views/app/component.spec.ts

```diff
@@ -1,8 +1,10 @@
 import { TestBed, async } from '@angular/core/testing';
-import {Title} from '@angular/platform-browser';
+import { Title } from '@angular/platform-browser';
```

Comment:
It looks like *.spec.ts and *.ts linters disagree on whether there should be whitespace around imported symbols. I don't really care, unless it's easy to fix.